phoenix-postgres-pgadmin/docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: '3.7' 
+ 
+services: 
+ 
+  example_app:
+    build: example_app
+    environment:  
+      - POSTGRES_USER=postgres 
+      - POSTGRES_PASSWORD=postgres 
+      - POSTGRES_DB=ExampleAppDB 
+      - PGHOST=db 
+      - PGPORT=5432 
+      - HOST=localhost
+      - PORT=4000
+    depends_on: 
+      - db 
+    ports:  
+      - "4000:4000" 
+
+  db: 
+    image: postgres 
+    restart: always 
+    ports: 
+    - "5432:5432" 
+    environment: 
+      POSTGRES_USER: postgres 
+      POSTGRES_PASSWORD: postgres 
+      POSTGRES_DB: ExampleAppDB 
+      PGDATA: /data/postgres 
+    volumes: 
+      - postgres:/data/postgres 
+    depends_on:  
+      - pgadmin 
+       
+  pgadmin: 
+      image: dpage/pgadmin4 
+      restart: always 
+      ports: 
+        - "5050:80" 
+      environment: 
+        PGADMIN_DEFAULT_EMAIL: pgadmin4@pgadmin.org 
+        PGADMIN_DEFAULT_PASSWORD: admin 
+      volumes: 
+        - pgadmin:/root/.pgadmin 
+   
+networks:
+  default:
+    ipam:
+      driver: default
+      config:
+      - subnet: 172.31.0.0/16
+
+volumes: 
+    pgadmin: 
+    postgres: 
+        
```

Comment:
Nit : add a new line at the end of the file

Reply:
added